src/test/java/kitchenpos/application/OrderServiceTest.java

```diff
@@ -34,7 +40,7 @@ class OrderServiceTest {
     @Mock
     private OrderRepository orderRepository;
     @Mock
-    private OrderTableRepository orderTableRepository;
+    private OrderValidatorImpl orderValidatorImpl;
```

Comment:
```suggestion
    private OrderValidator orderValidator;
```

인터페이스로 초기화하는 습관을 갖는게 좋습니다. 그런데 인터페이스가 필요한 클래스일까요?

Reply:
패키지를 묶어놔서 양방향 의존성이 생기지 않아 필요없을거같습니다!